### PR TITLE
Update Test Adapter for dev17

### PR DESCRIPTION
**Pull request by spebl**

### CODE REVIEW

File: GoogleTestAdapter/Packaging.TAfGT/source.extension.vsixmanifest [hunk 1]
```diff
@@ -13,10 +13,10 @@
         <Tags>Unit Test, C++, Google Test, GoogleTest, GTest, Test Explorer, Testing</Tags>
     </Metadata>
     <Installation InstalledByMsi="false">
-        <InstallationTarget Version="[15.0,17.0)" Id="Microsoft.VisualStudio.Pro" />
-        <InstallationTarget Version="[15.0,17.0)" Id="Microsoft.VisualStudio.Enterprise" />
-        <InstallationTarget Version="[15.0,17.0)" Id="Microsoft.VisualStudio.Ultimate" />
-        <InstallationTarget Version="[15.0,17.0)" Id="Microsoft.VisualStudio.Community" />
+        <InstallationTarget Version="[15.0,)" Id="Microsoft.VisualStudio.Pro" />
```
Comment (tristanlabelle) [inline GoogleTestAdapter/Packaging.TAfGT/source.extension.vsixmanifest]: Is the new version built still going to be compatible with Dev15?

**Review by tristanlabelle — DISMISSED** [review]

File: GoogleTestAdapter/Packaging.TAfGT/source.extension.vsixmanifest (hunk 1 above)
Comment (spebl) [reply GoogleTestAdapter/Packaging.TAfGT/source.extension.vsixmanifest]: Good call, doesn't look like it's compatible with pre-17. I'll update this.

**Review by tristanlabelle — APPROVED** [review]

File: GoogleTestAdapter/Packaging.TAfGT/source.extension.vsixmanifest [hunk 2]
```diff
@@ -36,9 +36,9 @@
         <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="Project" d:ProjectName="VsPackage.TAfGT" Path="|VsPackage.TAfGT|" />
     </Assets>
     <Prerequisites>
-        <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.8.27729.1,17.0)" DisplayName="Visual Studio core editor" />
-        <Prerequisite Id="Microsoft.VisualStudio.Component.VC.CoreIde" Version="[15.6.27406.0,17.0)" DisplayName="Visual Studio C++ core features" />
-        <Prerequisite Id="Microsoft.VisualStudio.Component.NuGet" Version="[15.9.28016.0,17.0)" DisplayName="NuGet package manager" />
+        <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.8.27729.1,)" DisplayName="Visual Studio core editor" />
```
Comment (Codiferous) [inline GoogleTestAdapter/Packaging.TAfGT/source.extension.vsixmanifest:39]: I don't understand the meaning of these very well, so I just wanted to call out that these reference 15.8. I'm not sure if they need to likewise be updated to 17.0?

**Review by Codiferous — APPROVED** [review]

File: GoogleTestAdapter/Packaging.TAfGT/source.extension.vsixmanifest (hunk 2 above)
Comment (spebl) [reply GoogleTestAdapter/Packaging.TAfGT/source.extension.vsixmanifest:39]: I think we're okay if we always limit the version we're available to install for. I think they probably could be updated, but I think it's just as safe to leave them as is.

**Pull request merged**